.env.test

```diff
@@ -60,7 +60,6 @@ L1_CORE_CONTRACT_ADDRESS="0xE2Bb56ee936fd6433DC0F6e7e3b8365C906AA057"
 
 ##### SNOS #####
 ## This is needed right now because Madara doesn't support getProof
-RPC_FOR_SNOS=""
```

Comment:
this is actually needed. it's just that it's being added in the CI right now so you don't see it failing

Reply:
Added back